src/api/v1/server.go

```diff
@@ -0,0 +1,11 @@
+package v1
+
+import (
+	"github.com/labstack/echo"
+)
+
+// TODO API Doc
+func API(api *echo.Group) {
+	api.GET("/motd", getMotd)
```

Comment:
a couple minutes of cache wouldn't hurt here, no? 

currently the requests to github are coming in from every client ip, if we switch to it all coming from one IP in AWS then that might look a lil suspicious xD

Reply:
good point. Again this is a temporary implementation, long term motd would be in a db table somewhere, and staff impactaccounts would have write access. Will add some cache

Reply:
fair point,  but IMO there still should be some cache on any close-to-static endpoint. even if the only downside were server load + database load on our end, there's still the downside of Making The Client Slower To Launch =)

if cloudflare can cache it, for me for example, my RTT latency to cloudflare is 9 milliseconds,  while through cloudflare to our heroku dyno, it's like 200 or 300.

on top of this, given that i already wrote the code in the `cloudflare` package to give the server the ability to invalidate cloudflare's cache, it's good to remember that we can do things like telling cloudflare to cache /v1/motd indefinitely.......  but make the api request to cloudflare to purge its cache whenever someone goes to their internal api or whatever to change the motd 😹

Reply:
> but make the api request to cloudflare to purge its cache whenever someone goes to their internal api or whatever to change the motd joy_cat

epic